examples/mnist/main.py

```diff
@@ -13,14 +13,20 @@
 
 
 class Net(nn.Module):
-    def __init__(self):
+    def __init__(self, num_local_experts):
```

Comment:
move moe example into separate example, for example `examples/mnist-moe`

Reply:
done